### DISCUSSION TITLE
Unable to add subdomains

### DISCUSSION THREAD

**Discussion by lete114** [deta/cloud-docs#491] [Help] [answered]
Just yesterday, I couldn't add a subdomain, I thought something happened to the deta server, so I waited until today to try again and still couldn't add a subdomain, checked the status code returned by the browser console is 503

**Comment by lete114** [comment]
I just realized that my question is consistent with [#485](https://github.com/orgs/deta/discussions/485)

**Reply by AyushSehrawat** [reply]
Not quite it's accurate :).... You can use custom domains if you have any for the time being

**Reply by lete114** [reply]
> Not quite it's accurate :).... You can use custom domains if you have any for the time being

The title is a subdomain, not a custom domain :)

**Reply by AyushSehrawat** [reply]
I meant to say since subdomains are disabled rn, you can use custom domains if you got any /shrug... Or wait for them to implement something on subdomains to avoid abuse

**Comment by abdelhai** [comment, marked as answer]
custom subdomains are have been re-enabled today
